Makefile

```diff
@@ -1,5 +1,5 @@
-PACKAGE_NAME=osbuild
-VERSION=1
```

Comment:
This is just to avoid hardcoding the version several places, right, or does this make a difference for packit in some way?

Reply:
just to avoid hardcoding